crates/net/downloaders/src/bodies/bodies.rs

```diff
@@ -329,18 +329,19 @@ where
 
         // Check if the provided range is the next expected range.
         let is_next_consecutive_range = *range.start() == *self.download_range.end() + 1;
+        let total_distance = *range.end() - *range.start();
         if is_next_consecutive_range {
             // New range received.
             tracing::trace!(target: "downloaders::bodies", ?range, "New download range set");
-            info!(target: "downloaders::bodies", "Downloading bodies {range:?}");
+            info!(target: "downloaders::bodies", "Downloading bodies {range:?} (total distance: {total_distance})");
```

Comment:
`distance` should be sufficient
```suggestion
            info!(target: "downloaders::bodies", "Downloading bodies {range:?} (distance: {total_distance})");
```